src/lang/simplified_chinese.lng

```diff
@@ -1,34 +1,34 @@
 ##grflangid 0x56
 
-STR_GRF_NAME                        :{TITLE} {VERSION}
-STR_GRF_DESC                        :{ORANGE}真实工业发展，绑定村庄人口与工业关系。{BLACK}自行添加脚本: 建议与村庄生长脚本一同使用，例如:Renewed Village Growth.{}{}{ORANGE}访问主页查看更多{}{}{BLACK}Created by 2TallTyler。
+STR_GRF_NAME                        :ITL 改进城镇工业 {VERSION}
```

Comment:
If you remove the {TITLE} string code, the name of the GRF in the game will not match the title in the in-game downloader. I'm not saying this is wrong, but other translations all keep the title un-translated.

Reply:
I have some reasons for doing so:
- Most players who use Chinese don't really read the titles of GRFs if they are not in Chinese, unless they are abbreviations or combinations of easy words, say, "Iron Horse," for example.
- Giving the GRF a translated name gives it the possibility to be more well-known among players and also reduces time to memorize.
- Given the fact that the whole Chinese modding community now likes translated names more, I personally would say that changing the title to a translated name would be a better choice.

Also, to avoid the condition where the player might not find the original name of the GRF, I've added the original title to the description.